doc/development.md

```diff
@@ -35,21 +35,21 @@ Create virtual environment for vdsm:
 
 ## Building Vdsm
 
-To configure sources (create Makefiles and configure script): 
+To configure sources (create Makefiles and configure script):
 
     git clean -xfd
     ./autogen.sh --system --enable-timestamp
 
-(It is recommended to start fresh after an update, from the previous step, before
-building RPMs, because the generated Makefile also contains the release number.)
+**_NOTE:_** It is recommended to start fresh after an update, from the previous step, before
+building RPMs, because the generated Makefile also contains the release number.
 
 Now you can create the virtual environment (https://docs.python.org/3/library/venv.html),
 which is necessary to run the tests later. This needs to be done only once:
 
     make venv
 
 To see configuration options run `./configure -h`. Configure things, if necessary,
-then start make to build Vdsm:
+then build Vdsm:
 
     make
 
```

Comment:
The changes in this commit should be squashed into the first commit if we need them.

Reply:
I squashed them all.